docs/usage/files.md

```diff
@@ -761,3 +762,57 @@ for instance in file_metadata:
 ```
 
 [get_all_metadata]: https://box-python-sdk.readthedocs.io/en/latest/boxsdk.object.html#boxsdk.object.item.Item.get_all_metadata
+
+Set a Classification
+--------------------
+
+To add classification to a [`File`][file_class], call [`file.set_classification(classification)`][set_classification].
+This method returns the classification type on the [`File`][file_class] object. If a classification already exists then 
+this call will update the existing classification with the new [`ClassificationType`][classification_type_class].
+
+```python
+from boxsdk.object.item import ClassificationType
+
+classification = client.file(file_id='11111').set_classification(ClassificationType.PUBLIC)
+print('Classification Type is: '.format(classification))
+```
+
+The set method will always work no matter the state your [`File`][file_class] is in. For cases already where a
+classification value already exists [`set_classification(classification)`][set_classification] may make multiple 
+API calls. 
+
+Alternatively, if you already know you have a classification and you are simple updating it, you can use the 
+[`update_classification(classification)`][update_classification]. This will ultimately help you save one extra API call.
+
+```python
+classification = client.file(file_id='11111').update_classification(ClassificationType.NONE)
+print('Classification Type is: '.format(classification))
+```
+
+[add_classification]: https://box-python-sdk.readthedocs.io/en/latest/boxsdk.object.html#boxsdk.object.item.Item.set_classification
```

Comment:
I think this reference name is mismatched with the docs above — it says add, but points to set